### PR TITLE
Feature 2

### PR DESCRIPTION
**Pull request by frykiz**

### CODE REVIEW

File: main.java [hunk 1]
```diff
@@ -16,6 +16,8 @@ public static void main(String args[]) {
                     System.out.println("\"" + pattern + "\"" + " found at line: " + counter);
                 }
                 counter ++;
+
```
Comment (martinhantom) [inline main.java:19]: Remove!

File: main.java [hunk 2]
```diff
@@ -16,6 +16,8 @@ public static void main(String args[]) {
                     System.out.println("\"" + pattern + "\"" + " found at line: " + counter);
                 }
                 counter ++;
+                
+                //test
```
Comment (martinhantom) [inline main.java:20]: Remove also!

**Review by martinhantom — CHANGES_REQUESTED** [review]
Remove the two lines!

File: main.java (hunk 2 above)
Comment (frykiz) [reply main.java]: Done

**Review by martinhantom — APPROVED** [review]
Swerve.

**Pull request merged**